q2_demux/_demux.py

```diff
@@ -48,6 +52,47 @@ def __next__(self):
         return next(self.barcode_generator), next(self.sequence_generator)
 
 
+def summary(output_dir: str, data: SingleLanePerSampleSingleEndFastqDirFmt) \
+        -> None:
+    per_sample_fastqs = list(data.sequences.iter_views(FastqGzFormat))
+    per_sample_fastq_counts = {}
+    for per_sample_fastq in per_sample_fastqs:
+        seqs = skbio.io.read(per_sample_fastq[1].open(),
+                             format='fastq',
+                             phred_offset=33, compression='gzip',
```

Comment:
Can you add a TODO comment about (possibly) reading the phred offset from the YAML file? Or we could add a phred offset attribute to the dir format in q2-types that could be used here.